quotemachine/app.js

```diff
@@ -1,101 +1,41 @@
-        
-        for(let i=0;i<randomColor.length;i++){
-            randomColor[i].style.backgroundColor="grey"
-        }
-        
-        document.body.appendChild(quote.innerHTML);
-    }
-    else if (0.4<randomColorText <= 0.5) {
-        quote.innerHTML = "Normally I miss deadlines like a storm trooper misses Jedi."
-        
-        for(let i=0;i<randomColor.length;i++){
-            randomColor[i].style.backgroundColor="orange"
-        }
-        
-        document.body.appendChild(quote.innerHTML);
-    }
-    else if (0.5<randomColorText <= 0.6) {
-        quote.innerHTML = "There's nothing an artist needs more - even more than excellent tools and stamina - than a deadline."
-        
-        for(let i=0;i<randomColor.length;i++){
-            randomColor[i].style.backgroundColor="brown"
-        }
-        
-        document.body.appendChild(quote.innerHTML);
-    }
-    else if (0.6<randomColorText <= 0.7) {
-        quote.innerHTML = "There’s no getting out alive, but you hope to avoid a deadline."
-        
-        for(let i=0;i<randomColor.length;i++){
-            randomColor[i].style.backgroundColor="DarkSlateBlue"
-        }
-        
-        document.body.appendChild(quote.innerHTML);
-    }
-    else if (0.7<randomColorText <= 0.8) {
-        quote.innerHTML = "Respect your deadline like it’s a field marshal."
-        
-        for(let i=0;i<randomColor.length;i++){
-            randomColor[i].style.backgroundColor="Fuchsia"
-        }
-        
-        document.body.appendChild(quote.innerHTML);
-    }
-    else if (0.8<randomColorText <= 0.9) {
-        quote.innerHTML = "Deadline-induced quality issues: a problem all over the galaxy."
-        
-        for(let i=0;i<randomColor.length;i++){
-            randomColor[i].style.backgroundColor="MediumSlateBlue"
-        }
-        
-        document.body.appendChild(quote.innerHTML);
-    }
-    else if (0.9<randomColorText <= 1) {
-        quote.innerHTML = "I do however like to tell people, especially regarding writing and deadlines: Don't be perfect, just be done."
-        
-        for(let i=0;i<randomColor.length;i++){
-            randomColor[i].style.backgroundColor="Salmon"
-        }
-        
-        document.body.appendChild(quote.innerHTML);
-    }
-})
 
 
+
+const newQuoteButton = document.getElementById("new-quote");
+let quote = document.getElementById("quote-text");
+const randomColor = document.getElementsByClassName("random-color");
+const randomTextColor = document.querySelector("#quote-text");
+
+newQuoteButton.addEventListener("click", function() {
+  let randomColorTextforR = Math.floor(Math.random() * 256);
+  let randomColorTextforG = Math.floor(Math.random() * 256);
+  let randomColorTextforB = Math.floor(Math.random() * 256);
+  let randomRGB = "rgb(" + randomColorTextforR + "," + randomColorTextforG + "," + randomColorTextforB + ")";
+
+  const quoteText = [
```

Comment:
I think quoteText (I'd use "quoteTexts" word as it holds an array of strings) should be outside current function (`newQuoteButton.addEventListener("click", function() {` ), in other words, for this homework you can put it to the global scope.